internal/aws/event/testdata/s3/created_single.json

```diff
@@ -27,7 +27,7 @@
                     "arn": "arn:aws:s3:::s3eventreceiver-dev"
                 },
                 "object": {
-                    "key": "test2.txt",
+                    "key": "year%3D2025/month%3D05/day%3D01/hour%3D10/minute%3D32/logs_778496226.json",
```

Comment:
This is what the key looks like when you read the SQS message without reversing the escaping. This key will not work when we call "get object".